taipy/gui/builder/_api_generator.py

```diff
@@ -83,6 +83,13 @@ def add_library(self, library: "ElementLibrary"):
                     element_name, f"{library_name}.{element_name}", element.default_attribute
                 ),
             )
+            # Allow element to be accessed from the root module
+            if hasattr(self.__module, element_name):
+                _TaipyLogger()._get_logger().info(
+                    f"Can't add element `{element_name}` of library `{library_name}` to the root of Builder API as it has already existed."  # noqa: E501
```

Comment:
Another element with the same name already exists

Reply:
just thought of it also. Thanks for the suggestion